### PR TITLE
Update the request re-use example

### PR DESCRIPTION
**Pull request by Tabrizian**
https://github.com/triton-inference-server/core/pull/258

### CODE REVIEW

File: src/simple.cc [hunk 1]
```diff
@@ -902,6 +906,20 @@ main(int argc, char** argv)
             InferResponseComplete, reinterpret_cast<void*>(p)),
         "setting response callback");
 
+    // When re-using the requests, we need to make sure that the request
```
Comment (nnshah1) [inline src/simple.cc:909]:
```suggestion
    // We need to make sure that the previous request was released before reusing it.
```

**Review by nnshah1 — APPROVED** [review]
slight tweak to comment for readability

**Review by rmccorm4 — DISMISSED** [review]

File: src/simple.cc (hunk 1 above)
Comment (Tabrizian) [reply src/simple.cc]: fixed.

**Review by rmccorm4 — APPROVED** [review]
:rocket:

**Pull request merged**